go/vt/topo/keyspace.go

```diff
@@ -36,6 +39,12 @@ import (
 
 // This file contains keyspace utility functions
 
+// Default concurrency to use in order to avoid overhwelming the topo server.
+// This uses a heuristic based on the number of vCPUs available -- where it's
+// assumed that as larger machines are used for Vitess deployments they will
+// be able to do more concurrently.
+var DefaultConcurrency = runtime.NumCPU()
```

Comment:
We are defaulting this to 32 for `GetTablets` so we now have two separate defaults depending on whether we are getting tablets or shards. The reasoning for 32 is covered by this comment. https://github.com/vitessio/vitess/pull/14693#discussion_r1417863256
I'm not completely opposed to changing the default for `GetTablets` to this value, but I'd prefer to change this 32 for the reasons listed in that comment.

Reply:
OK, I see that you were basing it on `fs.Int64Var(&topoReadConcurrency, "topo_read_concurrency", 32, "Concurrency of topo reads."), but you didn't reference the value. I'll see if I can use the variable somehow in both places.

Reply:
I addressed both of your comments here: https://github.com/vitessio/vitess/pull/15047/commits/c7a27d0b60d25421cb8e6185a4b32b67f758cc83